docs/metrics/rouge.md

```diff
@@ -0,0 +1,93 @@
+---
+search:
+  exclude: true
+---
+
+# ROUGE-N
+
+!!! info inline end "ROUGE vs. Recall"
+
+    Complimentary to BLEU, ROUGE-N can be thought of as an analog to [recall](recall.md) for text comparisons.
+
+ROUGE-N (**R**ecall-**O**riented **U**nderstudy for **G**isting **E**valuation), a metric within the broader ROUGE metric collection, is a vital metric in the field of natural language processing and text evaluation. It assesses the quality of a candidate text by measuring the overlap of n-grams between the candidate text and reference texts. ROUGE-N provides insights into the ability of a system to capture essential content and linguistic nuances, making it an important and versatile tool used in many NLP workflows. As the name implies, it is a recall-based metric - a complement to the precision-based BLEU score.
```

Comment:
Another ROUGE that we use on our text summarization workflow is `ROUGE-L`.  Aside from ROUGE-N are there any other variation of ROUGE score that we should include in the doc?

Reply:
There are also ROUGE-L, ROUGE-W, and ROUGE-S in the ROUGE collection - I was thinking to create a separate doc for them since ROUGE-L seems to be conceptually slightly different (there is also a recall and f1 -based component to ROUGE-L) and would result in a long megadoc. I can include all four metrics in the doc and try to keep it on the shorter side!

Reply:
We can comment on them in this doc but maybe have a separate doc for them and link it to this doc?

Reply:
Do you think ROUGE-L, W, and S can be summarized onto a single doc?  Should we create a linear ticket for this?